CHANGES.rst

```diff
@@ -45,6 +45,8 @@
   alignment boxes from slitmask design.
 - Added a table with DEIMOS slitmask design and objects info to the
   SlitTraceSet datamodel
+- Incorporates a procedure to enable GMOS N-S observations
```

Comment:
N-S is ambiguous given North/South. Can you write out nod and shuffle.

Reply:
done